modules/debts_api/app/sidekiq/debts_api/v0/form5655/vha/vbs_submission_job.rb

```diff
@@ -12,14 +12,21 @@ class V0::Form5655::VHA::VBSSubmissionJob
 
     class MissingUserAttributesError < StandardError; end
 
-    sidekiq_retries_exhausted do |job, _ex|
+    sidekiq_retries_exhausted do |job, ex|
       StatsD.increment("#{STATS_KEY}.failure") # Deprecate this in favor of exhausted naming convention below
       StatsD.increment("#{STATS_KEY}.retries_exhausted")
       submission_id = job['args'][0]
       user_uuid = job['args'][1]
-      UserProfileAttributes.find(user_uuid)&.destroy
+
       submission = DebtsApi::V0::Form5655Submission.find(submission_id)
-      submission.register_failure("VBS Submission Failed: #{job['error_message']}.")
+      submission&.register_failure("VBS Submission Failed: #{ex.message}")
```

Comment:
Small nit. 

Here, we are using the safe navigation thing (`&`) which is cool.

However, we are using the `find` method on the line above. Since we're using that, we can remove the safe navigation thing. Reason being is that if the `find` method can't find the submission, it will throw an exception.

You may have found this code as an example and I didn't realize this fact until now.